faults.csv

```diff
@@ -1,63 +1,70 @@
+NRP,NIRP or individual SSA interlock,NIRP or other individual SSA interlocks like PPS,"If the realtime status is OK (check legend), reset interlocks. If not, contact NIRP system experts.",SSA,PYDM,$PYDM/nirp/nirp_Lband_diagnostic_graphical.ui,"{""AREA"": {LINAC}, ""CM"": {CRYOMODULE}, ""RFNAME"": ""CM{CRYOMODULE}""}",,ACCL:{LINAC}:{CRYOMODULE}{CAVITY}0:SSA:,NRP_PRMT,1,,2,NIRP or individual SSA interlock,
+SSP,SSA Permit,SSA permit to turn on faulted,Run auto setup,CAV,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=4,SELCHAR=3",,ACCL:{LINAC}:{CRYOMODULE}{CAVITY}0:,SSA_LTCH,0,,2,SSA Permit,
+SSA,SSA Faulted,SSA not on,Run auto setup,SSA,EDM,$EDM/llrf/rf_srf_ssa_{cm_OR_hl}.edl,,,ACCL:{LINAC}:{CRYOMODULE}{CAVITY}0:SSA:,FaultSummary.SEVR,,2,2,SSA Faulted,
+BCS,BCS LLRF Drive Fault,BCS fault is interrupting LLRF drive (only affects CM01 in practice),Communicate the fault to the EOIC and await resolution,CM,EDM,$EDM/bcs/ops_lcls2_bcs_main.edl,,,ACCL:{LINAC}:{CRYOMODULE}00:,BCSDRVSUM,0,,2,BCS LLRF Drive Fault,
+FRO,LLRF Frequency Offset,"LLRF Frequency is offset from nominal because LLRF has shifted the frequency, or in frequency scan mode",Confirm that this isn't being actively used intentionally and then Rack Larger Range Freq Scan > Shift > Stop,RACK,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=1,SELCHAR=2",A,ACCL:{LINAC}:{CRYOMODULE}00:RACK{RACK}:,FREQSUM,,2,2,,
+FRO,LLRF Frequency Offset,"LLRF Frequency is offset from nominal because LLRF has shifted the frequency, or in frequency scan mode",Confirm that this isn't being actively used intentionally and then Rack Larger Range Freq Scan > Shift > Stop,RACK,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=1,SELCHAR=2",B,ACCL:{LINAC}:{CRYOMODULE}00:RACK{RACK}:,FREQSUM,,2,2,LLRF Frequency Offset,
+CSI,Cryo Summary Interlock,"CM Cryo Summary Interlock (He pressure, He Level, Cryo Heartbeat, and CP Ready) faulted","Check if the summary is OK and if yes, reset interlocks",CAV,EDM,$EDM/llrf/rf_srf_intlk_cryo.edl,,,ACCL:{LINAC}:{CRYOMODULE}{CAVITY}0:,CRYO_LTCH,0,,2,Cryo Summary Interlock,
+RLK,Resonance/Interlock Link,Resonance/ Interlock Link,Contact LLRF team,CAV,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=4,SELCHAR=3",,ACCL:{LINAC}:{CRYOMODULE}{CAVITY}0:,RESLINK_LTCH,0,,2,Resonance/Interlock Link,
+FPG,FPGA PLL Lock,The LLRF digital clock is no longer guranteed to be locked to the accelerator due to the FPGA detecting a temporary loss of externally supplied LO for the onboard clock,Contact LLRF team,CAV,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=4,SELCHAR=3",,ACCL:{LINAC}:{CRYOMODULE}{CAVITY}0:,PLL_LTCH,0,,2,FPGA PLL Lock,
+LOL,Local Oscillator Level,The LO (local oscillator) signal is used to generate clocks and for RF signal mixing in the RFS and PRC chassis. The system will not work properly if its amplitude is out of range.,CATER and if urgent contact AMRF,CAV,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=4,SELCHAR=3",,ACCL:{LINAC}:{CRYOMODULE}{CAVITY}0:,LO_LTCH,0,,2,LO Level,

```

Comment:
Can the last column of text be changed from "LO Level" to "Local Oscillator Level" just to make the fault decoder short description as obvious as possible?